Sources/Sentry/include/SentryFileManager.h

```diff
@@ -26,6 +26,8 @@ SENTRY_NO_INIT
 
 - (void)deleteAllFolders;
 
+- (NSArray<SentryFileContents *> *)getAllEvents;
+- (NSArray<SentryFileContents *> *)getAllEnvelopes;
```

Comment:
Are these two new methods going to replace `getAllStoredEventsAndEnvelopes`?

Reply:
Yep, my plan is to use them in `SentryHttpTransport`. I just kept it for now and wanted to do the change in another PR.